packages/client/src/editor/components/Stickies.tsx

```diff
@@ -7,69 +7,66 @@ import { Notes } from '@material-ui/icons';
 const UUID_NAMESPACE = '940beed9-f057-4088-a714-a9f5f2fc6052';
 
 interface prop {
-    get: (id: UUID) => string;
-    save: (content: string, id: UUID) => void;
-    remove: (id: UUID) => void;
+  get: (id: UUID) => string;
+  save: (content: string, id: UUID) => void;
+  remove: (id: UUID) => void;
 }
 
 export const notes: Note[] = [];
 
 export const addNote = (pos: Coordinate) => {
-    notes.push({
-      UUID: v5('line' + (notes.length-1).toString(), UUID_NAMESPACE),
-      position: pos,
-      text: 'empty note'
-    });
-    console.log('added note');
+  notes.push({
+    UUID: v5('line' + (notes.length - 1).toString(), UUID_NAMESPACE),
+    position: pos,
+    text: ''
+  });
+  console.log('added note');
 };
 
 const moveNote = (id: UUID, newPos: Coordinate) => {
-    let index = notes.findIndex( note => note.UUID == id);
-    if (index != -1) {
-      notes[index].position = newPos;
-    }
+  let index = notes.findIndex(note => note.UUID == id);
+  if (index != -1) {
+    notes[index].position = newPos;
+  }
 };
-  
+
 const updateNote = (content: string, id: UUID) => {
-    let index = notes.findIndex( note => note.UUID == id);
-    if (index != -1) {
-      notes[index].text = content;
-      console.log(content);
-    }
+  console.log('updating note', { id, content });
+  let index = notes.findIndex(note => note.UUID == id);
+  if (index != -1) {
+    notes[index].text = content;
+    console.log('updated note', { id, content });
+  }
 };
-  
+
 const deleteNote = (id: UUID) => {
-    let index = notes.findIndex( note => note.UUID == id);
-    if (index != -1) {
-      notes.splice(index,1);
-    }
+  let index = notes.findIndex(note => note.UUID == id);
+  if (index != -1) {
+    notes.splice(index, 1);
+  }
 };
 
 const getNote = (id: UUID) => {
-    let index = notes.findIndex( note => note.UUID == id);
-    if (index != -1) {
-      return notes[index].text;
-    } else return 'Note with ${id} not found'
+  let index = notes.findIndex(note => note.UUID == id);
+  if (index != -1) {
+    return notes[index].text;
+  } else return 'Note with ${id} not found';
 };
 
 const Stickies = () => {
-    const [notelist, addSticky] = useState(notes);
-    const listItems = notelist.map( note => {
-        <div key={note.UUID}>
-            {new StickyNote({
-                id: note.UUID,
-                get: getNote,
-                save: updateNote,
-                delete: deleteNote
-            })}
-        </div>
-    });
+  console.log({ notes });
+  const listItems = notes.map(note => (
+    <div key={note.UUID}>
+      <StickyNote
+        id={note.UUID}
+        get={getNote}
+        save={updateNote}
+        delete={deleteNote}
+      />
+    </div>
+  ));
 
-    return (
-        <div id='stickies'>
-            {notelist.map(note => <div key={note.UUID}>{listItems}</div>)}
```

Comment:
There was a bit of duplication here, since `listItems` already had all the notes, and here you render that list for every element of noteList (i.e. you get n^2 elements)